website/views/static/plans.jade

```diff
@@ -15,15 +15,15 @@ block content
       p
         =env.t('indivPlan1')
         |&nbsp;
-        a(href='http://habitica.wikia.com/wiki/Guilds' target='_blank')=env.t('guilds')
+        !=env.t('guildsLink')
         = ' ' + env.t('and') + ' '
-        a(href='http://habitica.wikia.com/wiki/Challenges' target='_blank')=env.t('challenges')
+        !=env.t('challengesLink')
         |&nbsp;
         =env.t('indivPlan2')
       p
         =env.t('groupText1')
         |&nbsp;
-        a(href='http://habitica.wikia.com/wiki/Habitica' target='_blank')=env.t('habitica')
+        !=env.t('habitica')
```

Comment:
I think you meant to use `habiticaLink` here.  `habitica` is still just the name of the site, and it is used [elsewhere in the code](https://github.com/HabitRPG/habitrpg/blob/77c276fefe85ea19a5fc6c596dc96395a5e7e0b3/website/views/index.jade#L5) so I suggest that doesn't change. :smile: